pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/OncHistoryDetail.java

```diff
@@ -433,6 +434,7 @@ public static Map<String, List<OncHistoryDetail>> getOncHistoryDetails(@NonNull
 
     public static Map<String, List<OncHistoryDetail>> getOncHistoryDetailsByParticipantIds(@NonNull String realm,
                                                                                            List<String> participantIds) {
+        logger.info("Getting onc histories for participants {}", Arrays.toString(participantIds.toArray()));
```

Comment:
I believe this can be simplified as 
        logger.info("Getting onc histories for participants {}", participantIds);

Reply:
Or `String.join(", ", participantIds);`?